### ISSUE TITLE
rstudio script's default_user.sh should be adjusted to support re-running

### ISSUE THREAD

**Issue by cboettig** [rocker-org/rocker-versioned2#83]
Currently it will error if rstudio user already exists.  This means that doing something nice and simple like 

```shell
RUN ROCKER_VERSION="daily" /rocker_scripts/install_rstudio.sh
```
to upgrade an existing installation will throw an error.  (also happens that we need to add `libpq5` for the newest daily).  

`default_user.sh` should check that user exists before creating `rstudio` user, etc.

**Comment by eitsupi** [comment]
Do you mean `default_user.sh`, not `userconf.sh`?

It was fixed.

https://github.com/rocker-org/rocker-versioned2/blob/cfac661b2871f59b31b8d56b3c6217cd6d521c5b/scripts/default_user.sh#L6-L22

**Comment by eitsupi** [comment]
Seems be closed by 3804e58e7323b35afd5a5d5097d8ebfa6e5ef4c2
